packages/migrations/src/actions/2024.12.02T00-00-00.legacy-user-org-cleanup.ts

```diff
@@ -0,0 +1,31 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2024.12.02T00-00-00.legacy-user-org-cleanup.ts',
+  async run({ sql, connection }) {
+    // Delete all the organizations owned by a legacy user
+    await connection.query(sql`
+      DELETE
+      FROM
+        "organizations"
+      WHERE
+        "user_id" = ANY(
+          SELECT
+            "id"
+          FROM
+            "users"
+          WHERE
+            "supertoken_user_id" IS NULL
+        )
```

Comment:
There is no cascade trigger for the `"organizations"."user_id"` foreign key constraint, so we first delete the organizations.